### DISCUSSION TITLE
Svelte / Vite unhappy with a reactive statement(?) in the sveltekit/supabase auth boilerplate

### DISCUSSION THREAD

**Discussion by 0xSharp** [supabase/supabase#18322] [Questions]
Hey guys,

as I tried to explain in the title, vite/svelte is getting upset with the +layout.svelte from supabase's tutorial for using auth with sveltekit. the snippet that's causing issues is

`$: ({ supabase, session } = data)`

It comes right after declaring supabase and session as a const in the line above and the compiler seems to think it's a declaration?

Has anyone run into this?

Thanks for any help/guidance!

**Comment by j4w8n** [comment]
Just use `let` in the line above, or I think you can actually remove the line above that.

**Reply by 0xSharp** [reply]
I'll give that a go then, thank you! Was kind of my first instinct, I just always get nervous messing with "tried and true" boilerplate. Figured I was doing something wrong.
